docs/deployment/deployK8s-AWS.md

```diff
@@ -5,343 +5,158 @@ title: Prepare AWS EKS for Multi-Region Deployments
+    * Click **Create transit gateway**.
 
-      * In the VPC Dashboard, select **Subnets**.
+    * Repeat this step in `us-east-1` region.
 
-      * For each subnet displayed, record the **Routing Table** value. You might have a single routing table for all of your subnets. You will use the routing table ID or IDs in a subsequent step.
+3.  Create the transit gateway peering connection attachment. Toggle to the `eu-west-1` region.
 
-      * At the top right of the page, change the region to **Oregon**, and repeat these steps to find and record the routing table ID or IDs for this VPC.
+    * Navigate to the **Transit gateway attachments** section and click **Create transit gateway attachments**.
 
-7. Modify the routing table or tables for each VPC to add a route to the other VPC using the peering connection that was created.
+    * Add a name tag such as `demo-peering-attachment-us-east-1`.
 
-      * In the VPC Dashboard, select **Route Tables** for the Oregon region.
+        > Make note that this name is refering to the region it is peering to, not the region it is being create in.
 
-      * Select the route table you recorded for the `us-west-2` (Oregon) VPC, and click the `Routes` button.
+    * Select the Transit gateway id that you just made in the `eu-west-1` region.
 
-         Two routes are displayed.
+    * Change **Attachment type** to `Peering Connection`.
 
-      * Click **Edit Routes** --> **Add Route**, and for **Destination**, enter the CIDR block for the `ca-central-1` cluster (172.16.0.0/16).
+    * For **Region** select `us-east-1`.
 
-      * For **Target**, select the VPC peering connection you created in a prior step. Click **Save Routes**.
+    * For **Transit gateway (accepter)** add the Transit gateway id that you just made in the `us-east-1` region.
 
-         A route for the `ca-central-1` cluster directed to the peering connection is displayed.
+    * Click **Create transit gateway attachment** .
 
-      * If more than one routing table is used for the `us-west-2` VPC, repeat the previous steps for each routing table.
+4.  Accept transit gateway peering attachment.
 
-      * At the top right of the page, change the region to **Canada Central**.
+    * Once the transit gateway peering connection shows `pending acceptance` as its **State**, toggle to the `us-east-1` region and select **Transit gateway attachments**.
 
-      * Select the route table you recorded for the `ca-central-1` (Canada Central) VPC, and click the `Routes` button.
+    * You should see the attachment you just made. Select **Actions** and click accept.
 
-         Two routes are displayed.
+    * Add a name to this attachment such as `demo-peering-attachment-eu-west-1`.
 
-      * Click **Edit Routes** --> **Add Route**, and for **Destination**, enter the CIDR block for the `us-west-2` cluster (10.0.0.0/16).
+        > Make note that this name is refering to the region it is peering to, not the region it is being create in.
```

Comment:
**_Note: This name refers to the region to which it is peering, not the region in which it is being created._**